common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -27,6 +27,12 @@ div.video {
     }
   }
 
+  &.is-poster {
```

Comment:
the label / naming convention of "is-poster" was a bit confusing to me, but I saw we use the word poster elsewhere in the code. I was expecting 'is-pre-roll" perhaps, but if that change were made it might be useful to apply everywhere. The smallest change I would suggest is a supporting comment above this line as shown below:
// CASE: video pre-roll state